main.tf

```diff
@@ -183,11 +183,12 @@ resource "aws_ecs_service" "lacework_datacollector" {
   name                = local.ecs_service_name
   cluster             = var.ecs_cluster_arn
   scheduling_strategy = "DAEMON"
+  launch_type         = "EC2"
```

Comment:
Should we make this configurable already?

Reply:
Yes, absolutely.  Currently the only launch types are EC2, FARGATE, and EXTERNAL - I was thinking that EC2 is the only one that applies to this module, but there's no guarantee AWS won't change that moving forward.  Thanks for calling it out!